pkg/crc/cache/cache_windows.go

```diff
@@ -23,4 +23,6 @@ func NewWin32BackgroundLauncherCache() *Cache {
 			return strings.TrimSpace(stdOut), nil
 		},
 	)
+	cache.ignoreNameMismatch = true
```

Comment:
Might be worth an explanatory comment? `upstream binary is win32-..., we want to name it crc-...`?